tests/processes/ProcessingTest.py

```diff
@@ -606,41 +607,36 @@ def check_prerequisites(cls):
     def test_global_instantiation(self):
         class TestOneBear(Bear):
 
-            def __init__(self, file_dict, section, queue, timeout=0.1,
-                         debugger=False):
+            def __init__(self, *args, **kwargs):
                 raise RuntimeError
 
-        class TestTwoBear(Bear):
+        class TestTwoBear(GlobalBear):
```

Comment:
I just made test more clear check the line `627` (just like a enhancement of test), it will also run fine without changing it to `GlobalBear`

Reply:
If the test runs fine without changing it to `GlobalBear`, please do not change the tests.
Create an issue and explain why you think it should be changed.
And it would probably be a newcomer issue.

Reply:
Sorry for the confusion please nvm the above comment, that change was meaningfull actually `GlobalBear` gets one more arg `file_dict` compare to other Bears. so I changed that to `GlobalBear` [here](https://github.com/coala/coala/pull/5677/files#diff-7c16cc0dc5d9e615bbb54239955f7b5eR625)

Reply:
I still dont get this.

You didnt change that in c1184ca58 .

So it isnt being 'undone' in this cleanup patch.

Is it 100% necessary?
Or are you doing an unrelated change?

Reply:
My guess is you need to revert to using `Bear.__init__(self, section, queue, timeout, debugger)` like before c1184ca